### PR TITLE
Reuse utils.ImportTagWithRetries in step/import_release

### PR DESCRIPTION
**Pull request by hongkailiu**
/cc @openshift/test-platform

### CODE REVIEW

File: pkg/steps/release/import_release.go [hunk 1]
```diff
@@ -107,54 +107,14 @@ func (s *importReleaseStep) run(ctx context.Context) error {
 	if err != nil {
 		return err
 	}
+	logrus.WithField("pullSpec", pullSpec).Debugf("Got the pull sepc")
```
Comment (deepsm007) [inline pkg/steps/release/import_release.go:110]: Typo “spec”

File: pkg/steps/release/import_release.go [hunk 2]
```diff
@@ -107,54 +107,14 @@ func (s *importReleaseStep) run(ctx context.Context) error {
...
-		if image.Image == nil {
-			return false, nil
-		}
-		pullSpec = streamImport.Status.Images[0].Image.DockerImageReference
-		return true, nil
-	}); err != nil {
+	if newPullSpec, err := utils.ImportTagWithRetries(ctx, s.client, s.jobSpec.Namespace(), "release", s.name, pullSpec); err != nil {
 		return fmt.Errorf("unable to import %s release image: %w", s.name, err)
+	} else {
+		logrus.WithField("newPullSpec", newPullSpec).Debugf("Got the new pull sepc")
```
Comment (deepsm007) [inline pkg/steps/release/import_release.go:116]: Same as above

Comment (hongkailiu) [reply pkg/steps/release/import_release.go]: Revised.

**Pull request merged**